tests/unit/test_schemas.py

```diff
@@ -85,8 +85,8 @@ def validate_node(tree, node, schema, label, name):
         pytest.fail(f"{label} {name} fails validation")
 
 
-def _tree_path(tree):
-    return os.path.relpath(os.path.abspath(tree._path))
+def _tree_path(tree: tmt.Tree):
+    return tree._path.absolute().relative_to(Path.cwd())
```

Comment:
I bet it can be simplified. Symlinks probably are not the biggest thread, path that's relative to some other directory when CWD is assumed by the code might be a problem, hence this "normalization".

Reply:
right, I was thinking adding the `.absolute()` to the tmt.Tree:
```
- self._path = path or Path.cwd()
+ self._path.absolute() = path or Path.cwd() 
```
but that's just useless overthinking from my part.

Reply:
Let's move on, we can always improve in the future :)